services/order.py

```diff
@@ -0,0 +1,40 @@
+from typing import Optional
+
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket, MovieSession
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: Optional[str] = None
+) -> None:
+    with transaction.atomic():
+        user = get_user_model().objects.get(username=username)
+        order = Order.objects.create(user=user)
+
+        if date:

```

Comment:
```suggestion
        if date is not None:

```

Reply:
Thx you! 
added contidion "is not None"